explorer/app/components/Project/common/projectTransformer.ts

```diff
@@ -15,6 +20,42 @@ export function buildProjectCitationPath(project?: ProjectResponse): string {
   return `explore/projects/${project.projects[0].projectId}`;
 }
 
+/**
+ * Builds project contributors from API response.
+ * @param project - Project response model return from API.
+ * @returns project contributors with their corresponding [organization] citation number.
+ */
+export function buildProjectContributors(
```

Comment:
I am seeing a different order of contributors between local and production. Can you take a look at this?

[Local](http://localhost:3000/explore/projects/60ea42e1-af49-42f5-8164-d641fdb696bc):
<img width="234" alt="Screen Shot 2022-06-28 at 12 25 21 PM" src="https://user-images.githubusercontent.com/2446424/176268300-4b50212a-9c10-4428-afec-7bf03baa9efa.png">

[Production](https://data.humancellatlas.org/explore/projects/60ea42e1-af49-42f5-8164-d641fdb696bc):
<img width="272" alt="Screen Shot 2022-06-28 at 12 25 45 PM" src="https://user-images.githubusercontent.com/2446424/176268368-29763253-ebdb-4ca5-9e5a-e07d2e05be49.png">

Reply:
It also looks like the role has fallen off? See `(Principal Investigator)` in the production screenshot above.

Reply:
The API returned lists the contributors in a different order.
Added the role - thanks for picking it up!